src/platform/platform_linux.cpp

```diff
@@ -49,16 +49,26 @@ std::string mp::platform::default_server_address()
     return {"unix:/run/multipass_socket"};
 }
 
-mp::VirtualMachineFactory::UPtr mp::platform::vm_backend(const mp::Path& data_dir)
+QString mp::platform::default_driver()
 {
-    auto driver = qgetenv("MULTIPASS_VM_DRIVER");
+    return QStringLiteral("qemu");
```

Comment:
We should think about transitioning from the environment variable to this setting for those that have previously used it.

This should probably be done in:
1) check if `MULTIPASS_VM_DRIVER` is set and use it as the initial value for the setting (for platforms other than snap)
2) `on-refresh` snap hook should `snapctl get driver`, `multipass set…` it and reset it (set to `~`, which is YAML's `null`?)
3) `configure` hook should fail and tell the user to use `multipass set…` for this key (I don't think supporting `snap set` along `multipass set` will do us good in the long run)

It doesn't all have to be this PR, as long as we have an upgrade story for both `edge` now and `beta` users when we release.

Reply:
OK. I'd create a separate issue for this, just for the sake of my organization. Are epochs already available? Otherwise, would we leave that transition code indefinitely?

Reply:
Right, we do need to do *something* here though, or people using `libvirt` on edge will get moved back to `qemu`, because we're not looking at the environment variable any more.

I think as a minimum we'd need to respect the environment variable as long as it's set, and once we implement migration we can warn that it is deprecated.

As for epochs… even if they are available, that'd only work for snaps.

Reply:
I *think* we should respect the environment variable if it's set, for now. Otherwise people using `snap set` will get ignored.

So, precedence:
1) `MULTIPASS_VM_DRIVER`
1) `multipass set local.driver`
1) platform default

Reply:
You're right, no way to check if there was an explicit configuration with the current interface.